src/_variables.css

```diff
@@ -4,8 +4,8 @@
 @import url('https://fonts.googleapis.com/css2?family=Bungee+Shade&family=Gravitas+One&display=swap');
 
 :root{
-  --background_color: ##F0C808;
-  --black_text: ##1A1A1A;
+  --background_color: #F0C808;
```

Comment:
Is this file creating CSS files usable by any part of the app?

Reply:
That is the idea, but I haven't been able to make use of the variable the way you just described it.